docs/antrea-ipam.md

```diff
@@ -46,3 +46,145 @@ cluster. Valid range is 16 to 30. Default is 24.
 
 - `nodeCIDRMaskSizeIPv6`: Mask size for IPv6 Node CIDR in IPv6 or dual-stack
 cluster. Valid range is 64 to 126. Default is 64.
+
+## Antrea Flexible-IPAM
+
+Since version 1.4, Antrea starts to support flexible control over Pod IP addressing. This
+can be achieved by configuring custom resource `IPPool` with desired set of IP ranges.
+The pool can be annotated to Namespace, Pod and PodTemplate of StatefulSet/Deployment.
+Antrea will manage IP address assignment for corresponding Pods according to pool spec.
+
+Note that IP pool annotation cannot be updated, but rather re-created. IP Pool can be
```

Comment:
IP pool annotation -> the IP pool annotation

What does "re-created" mean?

IP Pool -> "An IP pool" or "An IPPool"

Reply:
Fixed.
```
Note that the IP pool annotation cannot be updated or deleted without recreating the
resource. An `IPPool` can be extended, ...
```